tests/components/zwave_js/test_sensor.py

```diff
@@ -59,27 +54,27 @@ async def test_numeric_sensor(hass, multisensor_6, integration):
 
     assert state
     assert state.state == "9.0"
-    assert state.attributes["unit_of_measurement"] == TEMP_CELSIUS
-    assert state.attributes["device_class"] == DEVICE_CLASS_TEMPERATURE
+    assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == TEMP_CELSIUS
+    assert state.attributes[ATTR_DEVICE_CLASS] == SensorDeviceClass.TEMPERATURE
 
     state = hass.states.get(BATTERY_SENSOR)
 
     assert state
     assert state.state == "100.0"
-    assert state.attributes["unit_of_measurement"] == "%"
-    assert state.attributes["device_class"] == DEVICE_CLASS_BATTERY
+    assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == "%"
+    assert state.attributes[ATTR_DEVICE_CLASS] == BinarySensorDeviceClass.BATTERY
 
     ent_reg = er.async_get(hass)
     entity_entry = ent_reg.async_get(BATTERY_SENSOR)
     assert entity_entry
-    assert entity_entry.entity_category == ENTITY_CATEGORY_DIAGNOSTIC
+    assert entity_entry.entity_category == EntityCategory.DIAGNOSTIC
```

Comment:
Could you use `is` here rather than `==`?

Reply:
Agreed, done.